keyboards/metamechs/timberwolf/config.h

```diff
@@ -24,8 +24,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define PRODUCT_ID 0x5754 // TW
 #define DEVICE_VER 0x0001
 #define MANUFACTURER MetaMechs
-#define PRODUCT Timber Wolf
-#define DESCRIPTION Timber Wolf
+#define PRODUCT MetaMechs Timber Wolf
+#define DESCRIPTION MetaMechs Timber Wolf
```

Comment:
```suggestion
#define PRODUCT Timber Wolf
```

Reply:
I was trying to go by the VIA standard here: https://caniusevia.com/docs/configuring_qmk#change-product
is that outdated?

Reply:
I disagree with the recommendation there, as the `MANUFACTURER` string is where you should put your name, and `PRODUCT` where you put the name of the keyboard. The fact that some OSes don't show both is an issue with them not accommodating the USB spec (although admittedly it doesn't say much about the manu/product strings).